config/default.yml

```diff
@@ -1143,3 +1143,7 @@ Style/SymbolProc:
     - define_method
     - mail
     - respond_to
+
+Style/FormatStringToken:
```

Comment:
CI is failure:
https://app.circleci.com/pipelines/github/rubocop/rubocop-rails/2228/workflows/066ed396-8ff7-4bb2-af9d-1536fe51f602/jobs/14896

 Please move it above `Style/SymbolProc` setting.

Reply:
Thanks! I made that change. I noticed shortly after creating this PR though that the `IgnoredMethods` key is deprecated and that `AllowedMethods` is used instead. I updated the config to match what I saw used elsewhere. Let me know if that's ok or if there are more changes needed.

Reply:
@koic @javierjulio Why was `IgnoredMethods` replaced with `AllowedMethods`?  They have opposite meanings, and the documentation is also opposite.

The [documentation](https://www.rubydoc.info/gems/rubocop/1.22.1/RuboCop/Cop/IgnoredMethods) for `IgnoredMethods` was:

> This module encapsulates the ability to ignore certain methods when parsing

The [documentation](https://msp-greg.github.io/rubocop/RuboCop/Cop/AllowedMethods.html) for `AllowedMethods` is the opposite:

> This module encapsulates the ability to allow certain methods when parsing.

How are these interchangeable?

Reply:
I went off of other rules listed in the same file that have the same keys and "deprecated" notice. Since they listed both keys with the same list and since there wasn't explicit review follow up, I figured that was the right solution. If it's not though, it should be changed.

Reply:
It appears to be correct given the currect state of RuboCop, but it is unquestionably a bug.  There is no world where `Allowed` should be equivalent to `Ignored`.  So at this point it is an [unrelated bug](https://github.com/rubocop/rubocop/pull/10829#issuecomment-1402871729) in RuboCop Core, which if fixed will cause lots of problems. 😢

Reply:
Current behavior is correct, but the documentation confused me.

https://github.com/rubocop/rubocop/pull/10829#issuecomment-1403082579